api/src/main/java/org/openmrs/util/OpenmrsConstants.java

```diff
@@ -689,6 +694,10 @@ public static String getOpenmrsProperty(String property) {
 		props.add(new GlobalProperty(GP_MAIL_SMTP_STARTTLS_ENABLE, "false",
 		        "Set to true to enable TLS encryption, else set to false"));
 		
+		props.add(new GlobalProperty(GP_HOST_URL,
+		        "http://localhost:8080/openmrs/admin/users/changePassword.form?activationkey=",
```

Comment:
You can remove this default value for now, it will be added once we update the form